ppb/events.py

```diff
@@ -25,6 +26,7 @@
     'SceneStarted',
     'SceneStopped',
     'StopScene',
+    'TreeStructurePublisher',
```

Comment:
Minor quibble: `EventMixin` and `TreeStructurePublisher` should probably be at the top.